go.mod

```
@@ -18,23 +19,25 @@ require (
 	github.com/gravitational/trace v1.1.16-0.20210609220119-4855e69c89fc
 	github.com/hashicorp/go-version v1.2.1
 	github.com/jonboulle/clockwork v0.2.2
+	github.com/json-iterator/go v1.1.10
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/kr/text v0.2.0 // indirect
 	github.com/mailgun/holster/v3 v3.15.2
+	github.com/manifoldco/promptui v0.8.0
 	github.com/pelletier/go-toml v1.8.0
-	github.com/sergi/go-diff v1.1.0 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible
```

Looks like diskv has a proper go mod compatible version, can you update the imports to `github.com/peterbourgon/diskv/v3`?
```suggestion
	github.com/peterbourgon/diskv/v3 v3.0.0
```